src/context/AppContext.tsx

```diff
@@ -27,7 +27,7 @@ export const AppContext = createContext<AppContextValues>({});
 
 export const AppContextProvider = ({ children }: { children: ReactNode }) => {
   let initialWallet;
-  let initialUser;
+  const initialUser = null;
```

Comment:
Initial user should not be initialised to null, else it will never be `undefined` and thus there's no benefit to introducing the `null` type to the `User` object. The logic I implemented for the User wizard is now on master so a rebase should resolve.